### PR TITLE
TZQ variations

### PR DESCRIPTION
**Pull request by sihyunjeon**

### CODE REVIEW

**Comment by sihyunjeon** [comment]
@agrohsje to provide you some context, i had a call with carlos,

mg29x seems very slow with tzq process and also crashes.

we are testing it with variation in ptjet bin (maybe divergence? UL had ptj 0.1 cut) and also having tzq onshell as a backup

**Comment by sihyunjeon** [comment]
we should check if TZQ analysis utilizes b and q in their event selections since ptj5 might affect the cross section badly (not easily quantifiable i think)

File: Cards/MadGraph5_aMCatNLO/NewTests/TZQB-4FS_PTJ5_amcatnlo-pythia8/TZQB-4FS_PTJ5_amcatnlo-pythia8.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "model_params_user": [
+        "set run_card ptj 5."
+    ], 
+    "model_params_vars": {},
+    "fragment": ["Generator/ExternalLHEProducer_MadGraph5_aMCatNLO.dat", "PartonShower/amcatnlo-pythia8.dat"],
+    "fragment_user": [],
+    "fragment_vars": {
+        "processParameters": [
+            "'TimeShower:nPartonsInBorn = 3'",
+            "'TimeShower:mmaxgamma = 4'"
```
Comment (Cvico) [inline Cards/MadGraph5_aMCatNLO/NewTests/TZQB-4FS_PTJ5_amcatnlo-pythia8/TZQB-4FS_PTJ5_amcatnlo-pythia8.json:19]: mMaxGamma here. I think I fixed it in the ones under SingleT but not in here.

**Comment by agrohsje** [comment]
Shouldn't this be OnShell. 
I would keep the generators as in the list. Otherwise use capital letters for new words except "to"  for improved readability between particles and cuts which are all capitalized.

**Pull request merged**